templates/patterns/timeline/pattern-timeline.html.twig

```diff
@@ -0,0 +1,20 @@
+{#
+/**
+ * @file
+ * Timeline pattern.
+ *
+ */
+#}
+{% set _toggle_collapsed = toggle_collapsed|default('Display %d more'|t) %}
```

Comment:
I didn't mean for you to use a different formulation then in netlify, please revert this